app/javascript/dashboard/modules/widget-preview/components/Widget.vue

```diff
@@ -32,52 +76,182 @@ export default {
     isOnline: {
       type: Boolean,
       default: true,
     },
     replyTime: {
       type: String,
-      default: 'few minutes',
+      default: 'in_a_few_minutes',
+    },
+    color: {
+      type: String,
+      default: '#1f93ff',
+    },
+    widgetBubblePosition: {
+      type: String,
+      default: 'left',
+    },
+    widgetBubbleLauncherTitle: {
+      type: String,
+      default: 'Chat with us',
+    },
+    widgetBubbleType: {
+      type: String,
+      default: 'standard',
     },
   },
+  data() {
+    return {
+      widgetScreens: [
+        {
+          id: 'default',
+          title: this.$t('INBOX_MGMT.WIDGET_BUILDER.WIDGET_SCREEN.DEFAULT'),
+          checked: true,
+        },
+        {
+          id: 'chat',
+          title: this.$t('INBOX_MGMT.WIDGET_BUILDER.WIDGET_SCREEN.CHAT'),
+          checked: false,
+        },
+      ],
+      isDefaultScreen: true,
+      isWidgetVisible: true,
+    };
+  },
   computed: {
     getWidgetHeadConfig() {
       return {
         welcomeHeading: this.welcomeHeading,
-        welcomeTagLine: this.welcomeTagLine,
+        welcomeTagline: this.welcomeTagline,
         websiteName: this.websiteName,
-        websiteDomain: this.websiteDomain,
         logo: this.logo,
-        isExpanded: this.isExpanded,
+        isDefaultScreen: this.isDefaultScreen,
         isOnline: this.isOnline,
-        replyTime: this.replyTime,
+        replyTime: this.replyTimeText,
+        color: this.color,
       };
     },
+    getWidgetBodyConfig() {
+      return {
+        isDefaultScreen: this.isDefaultScreen,
+        isOnline: this.isOnline,
+        replyTime: this.replyTimeText,
+        color: this.color,
+      };
+    },
+    getWidgetFooterConfig() {
+      return {
+        isDefaultScreen: this.isDefaultScreen,
+        color: this.color,
+      };
+    },
+    replyTimeText() {
+      switch (this.replyTime) {
+        case 'in_a_few_minutes':
+          return this.$t(
+            'INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_FEW_MINUTES'
+          );
+        case 'in_a_few_hours':
+          return this.$t('INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_FEW_HOURS');
+        case 'in_a_day':
+          return this.$t('INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_DAY');
+        default:
+          return this.$t('INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_FEW_HOURS');
+      }
+    },
+  },
+  methods: {
+    handleScreenChange(item) {
+      this.isDefaultScreen = item.id === 'default';
+    },
+    toggleWidget() {
+      this.isWidgetVisible = !this.isWidgetVisible;
+      this.isDefaultScreen = true;
+    },
   },
 };
 </script>
 
 <style lang="scss" scoped>
-.text-lg {
-  font-size: var(--font-size-default);
+.screen-selector {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 .widget-wrapper {
-  box-shadow: var(--shadow-larger);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-shadow: 0 0px 20px 5px rgb(0 0 0 / 10%);
```

Comment:
We can use a box-shadow variable here

Reply:
@fayazara --shadow-large and --shadow-larger are very light towards the top. So I used a custom one. Could you please confirm which looks good, the custom or the variable shadow?

Reply:
Maybe we can just create a new variable for this - `--shadow-widget-builder` or something similar. @aswindevps

Reply:
Done